packages/babel-cli/src/babel/util.js

```diff
@@ -6,6 +7,7 @@ let util      = require("babel-core").util;
 let path      = require("path");
 let fs        = require("fs");
 let _         = require("lodash");
+let each      = require("lodash/each");
```

Comment:
Is this necessary? 
Lodash was called at line 9 so you should write `_.each` or `let each = _.each`